fe/fe-core/src/main/java/org/apache/doris/datasource/property/metastore/MetastoreProperties.java

```diff
@@ -45,6 +45,7 @@ public class MetastoreProperties extends ConnectionProperties {
 
     public enum Type {
         HMS("hms"),
+        ICEBERG("iceberg"),
```

Comment:
these type are not on the same level

Reply:
The type field accepts values such as Paimon, Iceberg, and HMS.